package mantis;

message block_header
{

	//state of block
	enum state_t
	{
        acquiring = 0;
        acquired = 1;
        writing = 2;
        written = 3;
	}
	
	required state_t state = 1;
	
	required uint64 acquisition_id = 2;
	
	required uint64 record_id = 3;
	
	required uint64 timestamp = 4;
	
	required uint32 data_size = 5;
	
}